itou/utils/templatetags/datetime_filters.py

```diff
@@ -21,3 +23,34 @@ def duration(value):
             return f"{minutes} min"
 
     return "N/A"
+
+
+@register.filter(expects_localtime=True)
+def naturaldate(value):
```

Comment:
Tu aurais pu essayer de te baser sur utiliser `django.utils.timeuntil` ou `django.utils.since` avec depth=1
Tu aura `X années` `X mois`  `X semaines` ou `X jours`.
Si cela convient aux UX, cela fait toujours ça de code en moins à maintenir :shrug:

Reply:
`timesince` (le filtre) ne correspond pas vraiment à ce qui est demandé (je n'ai pas l'impression qu'il y ait ce besoin de précision et qu'un langage naturel est souhaité.

Par contre j'ai vu que `django.utils.timesince` prend un paramètre `depth` qui pourrait être défini à `1`, simplifier un peu le code et le partager avec Django. En revanche ça nécessiterait toujours un filtre custom.

Je vais voir avec Zohra et Marion.

Reply:
Zohra [a confirmé](https://gip-inclusion.slack.com/archives/CU8ATF54L/p1741944129146899?thread_ts=1741943558.714919&cid=CU8ATF54L) l'approche 1.

Je ne suis pas vraiment satisfait de ce que fait timesince, qui au demeurant est un peu plus complexe pour notre besoin. Je suis plutôt d'avis de conserver quelque chose de simple, dont les permutations sont bien testées à coup de parametrize.